Policies, General Operations/Standard Protocol.md

```diff
@@ -2,26 +2,30 @@
 
 1. **Draft Phase**
 
-	During this phase, any Contributor may draft up a policy or change to a policy. They should gain the approval for policy or change from at least one (1) other Contributor before proceeding onto the next Phase.
+	During this phase, any Contributor may draft up a policy or change to a policy. They should gain the approval for policy or change from at least one (1) other Contributor before proceeding onto the next Phase
```

Comment:
Missing period at the end of the last sentence.

Reply:
fixed